packages/starknet-snap/src/chain/transaction-service.ts

```diff
@@ -116,13 +115,13 @@ export class TransactionService {
       transactionsOnChain.push(tx);
       transactionsOnChainSet.add(tx.txnHash);
     }
-
     for await (const tx of this.getTransactionsOnState(
       address,
       contractAddress,
     )) {
+      console.log(tx.txnHash);
```

Comment:
```suggestion
```

btw what is the reason to have normalizeStarknetHex? is it the response of the tx hash from provider is less than 66? which is different than the one that we insert into state?